qa/smoke-test-http/src/test/java/org/elasticsearch/http/snapshots/RestGetSnapshotsIT.java

```diff
@@ -50,6 +52,7 @@
     protected Settings nodeSettings(int nodeOrdinal, Settings otherSettings) {
         return Settings.builder()
             .put(super.nodeSettings(nodeOrdinal, otherSettings))
+            .put(LARGE_SNAPSHOT_POOL_SETTINGS)
```

Comment:
I'd prefer to just use a local settings object (class or inline) with the specific needs of this test (which I believe is 2?).

Reply:
I've moved LARGE_SNAPSHOT_POOL_SETTINGS to RestGetSnapshotsIT. I believe 2 is insufficient, the test hangs with 2 threads, but works with 3.